README.md

```diff
@@ -57,19 +57,15 @@ If either start_time and end_time are invalid or set to None, the simulation wil
 
 # Developer installation
 
-Clone the repository:
-`git clone https://github.com/urbanopt/urbanopt-ditto-reader.git`
-
-Change directories into the repository:
-`cd urbanopt-ditto-reader`
-
-If you are using an ARM chip (Apple Silicon) computer, install this branch of OpenDSSDirect.py:
-`pip install git+https://github.com/dss-extensions/OpenDSSDirect.py@dss_python-0.12.0`
-
-Install the respository:
-
-`pip install -e .`
-
+- Clone the repository: `git clone https://github.com/urbanopt/urbanopt-ditto-reader.git`
+- Change directories into the repository: `cd urbanopt-ditto-reader`
+- Create & activate your favorite venv in the repo (I prefer pyenv for [mac/linux](https://github.com/pyenv/pyenv#installation) or [windows](https://github.com/pyenv-win/pyenv-win#installation), and the [virtualenv plugin](https://github.com/pyenv/pyenv-virtualenv))
+    - Activate pre-commit (only once, after making a new venv): `pre-commit install`
+    - Runs automatically on your staged changes before every commit
+- Update pip and setuptools: `pip install -U pip setuptools .`
+- Install the respository with developer dependencies: `pip install -e .[dev]`
```

Comment:
Was there a way for us to get the install working without requiring the dev installation of `pip install -e .` ?

Reply:
Ahh this is just for the developer installation right?

Reply:
Correct, only developers should install with `.[dev]`. That installs additional dependencies (pytest, ruff, etc) that are only needed for development. Regular users who clone the repo should use `pip install .`

Reply:
They're defined in pyproject.toml under `[project.optional-dependencies]`